### PR TITLE
Manually verified contracts source link

### PR DESCRIPTION
**Pull request by aminlatifi**
Changed manuallyVerified json object
Defined schema for manuallyVerify.jsonc

### CODE REVIEW

File: packages/shared-pure/src/types/UrlPath.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { z } from 'zod'
+
+export const UrlPath = z
+  .string()
+  .refine((data) => /^(https?:\/\/[^\s/$.?#].[^\s]*)$/.test(data), {
```
Comment (antooni) [inline packages/shared-pure/src/types/UrlPath.ts:5]: Move this check to .schema.json

**Review by antooni — CHANGES_REQUESTED** [review]
Please either choose parsing via zod or validating via json schema

File: packages/config/src/verification/verifyManuallyVerifiedContracts.test.ts
Comment (antooni) [inline packages/config/src/verification/verifyManuallyVerifiedContracts.test.ts:1]: This file is not needed, the same test is int the manuallyVerifiedContracts.test.ts

File: packages/shared-pure/src/types/api/ManuallyVerifiedContrats.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+const ETHEREUM_ADDRESS_REGEX = /0x[a-fA-F0-9]{40}/
+
+const ethereumAddressSchema = z
+  .string()
+  .refine((address) => ETHEREUM_ADDRESS_REGEX.test(address), {
+    message: 'The key must be a valid Ethereum address',
+  })
+
+export const ManuallyVerifiedContracts = z.record(
+  ethereumAddressSchema,
```
Comment (antooni) [inline packages/shared-pure/src/types/api/ManuallyVerifiedContrats.ts:12]: `stringAs(EthereumAddress)`

Comment (aminlatifi) [reply packages/shared-pure/src/types/api/ManuallyVerifiedContrats.ts]: Okay, it's fixed now.

**Pull request merged**